### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nplasterer**
cc: @rsanheim

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 # Testing
 
 things
+
+
```
Comment (nplasterer) [inline README.md]: 👍 👎 💯 🔢 🎱 🌵 🍰 🐫

File: README.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+
+
+
```
Comment (nplasterer) [inline README.md]: another comment less emoji 🚯

File: README.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+going to cause conflicts
```
Comment (nplasterer) [inline README.md]: last one

File: README.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
 # Testing
```
Comment (rsanheim) [inline README.md]: test

File: README.md [hunk 5]
```diff
@@ -1,3 +1,10 @@
...
 
 things
+
```
Comment (rsanheim) [inline README.md]: another comment

File: README.md [hunk 6]
```diff
@@ -1,3 +1,10 @@
 # Testing
 
 things
```
Comment (rsanheim) [inline README.md]: another comment...

Comment (rsanheim) [inline README.md]: another comment

File: README.md [hunk 7]
```diff
@@ -1,3 +1,10 @@
...
+
+
+
```
Comment (rsanheim) [inline README.md]: test

File: README.md (hunk 1 above)
Comment (rsanheim) [inline README.md]: test

File: README.md (hunk 7 above)
Comment (rsanheim) [inline README.md]: adfad

**Comment by rsanheim** [comment]
hmm

**Pull request closed without merging**